metau-capstone/ScheduleViewController.m

```diff
@@ -39,4 +42,74 @@ - (void)prepareForSegue:(UIStoryboardSegue *)segue sender:(id)sender {
 }
 */
 
+- (nonnull __kindof UICollectionViewCell *)collectionView:(nonnull UICollectionView *)collectionView cellForItemAtIndexPath:(nonnull NSIndexPath *)indexPath {
+    NSNumber *cellNum = @(indexPath.row+1);
+    ScheduleCell *cell = [self.scheduleCollection dequeueReusableCellWithReuseIdentifier:@"ScheduleCollectionCell" forIndexPath:indexPath];
+    cell.dayNum.text = [NSString stringWithFormat:@"%ld", indexPath.row+1];
+    cell.layer.borderColor = [UIColor blackColor].CGColor;
+    cell.layer.borderWidth = 1;
+    
+    // Query for user's day
+    PFQuery *queryForDay = [PFUser query];
+    PFUser *const user = [PFUser currentUser];
+    [queryForDay getObjectInBackgroundWithId:user.objectId block:^(PFObject * _Nullable object, NSError * _Nullable error) {
```

Comment:
Something to think about --

The user and the user's current day doesn't ever change. It's global state for the entire view. Do we need to query it for every cell?

Reply:
Ah I see what you mean! I have refactored the code, removing the unnecessary queries for the user.